src/network/networkd-link.c

```diff
@@ -948,8 +948,6 @@ static int nexthop_handler(sd_netlink *rtnl, sd_netlink_message *m, Link *link)
 
         assert(link);
         assert(link->nexthop_messages > 0);
-        assert(IN_SET(link->state, LINK_STATE_CONFIGURING,
-                      LINK_STATE_FAILED, LINK_STATE_LINGER));
 
         link->nexthop_messages--;
```

Comment:
same here

Reply:
This change was dropped. This is not necessary to remove this assertion. Thanks.